cluster/rpc/client.go

```diff
@@ -188,23 +194,32 @@ func (cl *Client) getConn(leaderRaftAddr string) (*grpc.ClientConn, error) {
 	}
 
 	if cl.leaderRpcConn != nil {
-		// close open conn if leader addr changed
-		cl.leaderRpcConn.Close()
+		if err := cl.leaderRpcConn.Close(); err != nil {
+			cl.logger.WithFields(
+				logrus.Fields{
+					"error":       err,
+					"leader_addr": leaderRaftAddr,
```

Comment:
Q: in the previous `Client.Close()` we print `cl.leaderRaftAddr` as `leader_addr` and here we print `leaderRaftAddr` that is passed as arguments. Any difference between these two?

Because, in both cases, we trying to close `cl.leaderRpcConn`.

Reply:
good catch, this shall be `cl.leaderRaftAddr` given that the old one is closing, I am actually thinking that we print both

Reply:
updated to print both and be more descriptive